provision/ec2.go

```diff
@@ -210,17 +242,17 @@ func (p *EC2Provisioner) CreateInstance(opts EC2CreateInstanceOptions) (*cfg.Rem
 		},
 	})
 	if err != nil {
-		fmt.Fprintf(p.out, "Failed to set tags: %s", err.Error())
+		fmt.Fprintln(p.out, "Failed to set tags: "+err.Error())
```

Comment:
Wait I think the variable passing into string is better than direct concat. Does `fmt.Fprintln` not allow it or something?

Reply:
Unfortunately there's no `fmt.Fprintfln()` (probably for good reason, because thats a ridiculous function name)

I think it's because for here, I've decided to standardize on using concat if the attached variable is the last part of a string - there's a few example of the code where i use formatting directives to insert strings into the middle